bin/rat

```diff
@@ -20,6 +20,7 @@ parser.add_argument("--num_warmup", type = int, default = 1000, help = "Number o
 parser.add_argument("--num_draws", type = int, default = 1000, help = "Number of draws to sample after warmup (only available for sample)")
 parser.add_argument("--retries", type = int, default = 4, help = "Number of times to retry optimization (only available for optimize)")
 parser.add_argument("--tolerance", type = float, default = 1e-2, help = "All optimization results must be within rtol = atol = tolerance of each other or throw an error (only available for optimize)")
+parser.add_argument("--target_acceptance_rate", type = float, default = 0.85, help = "Target acceptance rate for adaptation")
```

Comment:
Along with updating the documentation I changed an argument to sample to allow setting the target accept rate